spring-pulsar-docs/src/main/asciidoc/quick-tour.adoc

```diff
@@ -25,26 +27,75 @@ The minimum supported versions for the underlying libraries required by the fram
 | 7.x (7.5 or later)
 |===
 
-=== Building the Project
+=== Quick Sample
 
-Spring for Apache Pulsar uses Gradle as it's build tool. Run the following command to do a full build of the project.
-[indent=0]
+In the following sample Spring Boot application, we show how to write a publisher and consumer using Spring for Apache Pulsar.
+This is a complete application and does not require any additional configuration as long as you have Pulsar cluster running on the default location - `localhost:6650`.
+
+NOTE: We recommend the usage of a Spring-Boot-First-Approach for Spring for Apache Pulsar based application as that simplifies things tremendously. To encourage this, a `spring-pulsar-spring-boot-starter` module is published that can easily be consumed by an application as a dependency.
+
+==== Dependencies
+
+Spring Boot applications only need the `spring-pulsar-spring-boot-starter` dependency. The following shows how to define the dependency for Maven and Gradle, respectively:
+
+[source,xml,indent=0,subs="verbatim,attributes",role="primary"]
```

Comment:
By doing `role="primary|secondar"` it automatically puts them into a tabbed pane in the rendered html - its awesome!